docs_sphinx/user/computation.rst

```diff
@@ -179,6 +174,58 @@ threads used in the simulation.
 
 .. _standalone_custom_build:
```

Comment:
This will associate the `standalone_custom_build` reference with the new "Custom code injection" section – it should move down to the respective section.